AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -344,21 +344,23 @@ class CallingContext: NSObject {
             }
             if let userIdentifier = remoteParticipant.identifier.stringValue,
                self.displayedRemoteParticipants.value(forKey: userIdentifier) != nil {
-                self.notifyRemoteParticipantIsMutedChanged()
+                self.notifyRemoteParticipantViewChanged()
             }
         }
 
         participantsEventsAdapter?.onIsSpeakingChanged = { [weak self] remoteParticipant in
             guard let self = self else {
                 return
             }
-            if remoteParticipant.isSpeaking,
-               let userIdentifier = remoteParticipant.identifier.stringValue {
-                // Swap in speaking participant if not currently displayed
+            if let userIdentifier = remoteParticipant.identifier.stringValue {
                 if self.displayedRemoteParticipants.value(forKey: userIdentifier) == nil {
-                    if self.displayedRemoteParticipants.count == CallingContext.remoteParticipantsDisplayed {
+                    // Swap in speaking participant if not currently displayed
+                    if remoteParticipant.isSpeaking,
```

Comment:
Would it make sense to combine these conditionals?
``` swift
if self.displayedRemoteParticipants.value(forKey: userIdentifier) == nil, 
  if remoteParticipant.isSpeaking {
    ...
}
```

Reply:
If we combine the conditionals in the `if` statements, we'll have to add another condition in the `else` statement:
```swift
if self.displayedRemoteParticipants.value(forKey: userIdentifier) == nil,
   remoteParticipant.isSpeaking,
   self.displayedRemoteParticipants.count == CallingContext.remoteParticipantsDisplayed {
    ...
} else if self.displayedRemoteParticipants.value(forKey: userIdentifier) != nil {
    ...
}
```
Either way works I guess, it's more of a preference among the team.

Reply:
I think the `else if` one has better readability for me

Reply:
Beside this, other code looks good to me